src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java

```diff
@@ -35,6 +35,7 @@
 import org.slf4j.LoggerFactory;
 
 import kafka.admin.AdminUtils;
+import kafka.admin.RackAwareMode;
```

Comment:
They keep changing the API in incompatible ways.  I don't think this parameter is important, there is only a single broker, so the default rack aware mode is fine.